src/components/presentational/AnimatedRing/AnimatedRing.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { CSSProperties } from 'react'
+import './AnimatedRing.css'
+
+export interface AnimatedRingProps {
+    children: React.ReactNode;
+    style?: CSSProperties;
+    color?: string;
```

Comment:
Ideally use ColorDefinition here (which was added recently), but not a big deal

Reply:
Done.